numpy/lib/_function_base_impl.py

```diff
@@ -986,7 +986,7 @@ def gradient(f, *varargs, axis=None, edge_order=1):
     ----------
     f : array_like
         An N-dimensional array containing samples of a scalar function.
-    varargs : list of scalar or array, optional
+    varargs : list of scalar or array, optional. (see Examples below).
```

Comment:
@ngoldbaum Add: see Examples below

Reply:
I don't think this renders very nicely in the built docs:

<img width="693" alt="image" src="https://github.com/numpy/numpy/assets/3126246/4102ca0e-8723-46c7-ab01-3fd1e1eb9a6e">

Can you either add the new "See examples below" suggestion either on the very first or very last line of the explanation below the name of the argument? I think that would read better.

Reply:
I copied the model from [polyfromroots](https://numpy.org/doc/stable/reference/generated/numpy.polynomial.polynomial.polyfromroots.html) at the end of the argument description and with parenthesis.

![image](https://github.com/numpy/numpy/assets/14118472/0fb8a7df-7c33-44e6-befe-54d0227355ee)

![image](https://github.com/numpy/numpy/assets/14118472/afba91de-f5d8-4161-b8fb-9d6cb704853c)